### PR TITLE
Add documentation about login page

### PR DESCRIPTION
**Pull request by djhi**

### CODE REVIEW

File: packages/ra-auth-cognito/Readme.md [hunk 1]
```diff
@@ -96,6 +101,71 @@ To support react-admin [identity feature](https://marmelab.com/react-admin/AuthP
...
+                    <CardMedia
+                        sx={{ height: 140 }}
+                        image="/login_background.jpg"
+                    />
+                    <CardContent>
+                        <LoginForm redirectTo="/" />
+                    </CardContent>
+				</Card>
+			</Box>
+		</>
```
Comment (fzaninotto) [inline packages/ra-auth-cognito/Readme.md:127]: strange indentation

File: packages/ra-auth-cognito/Readme.md [hunk 2]
```diff
@@ -96,6 +101,71 @@ To support react-admin [identity feature](https://marmelab.com/react-admin/AuthP
...
+                    />
+                    <CardContent>
+                        {requireNewPassword
+                            ? <PasswordSetupForm onSubmit={submit} />
+                            : <LoginForm onSubmit={submit} />
+                        }
+                    </CardContent>
+				</Card>
+			</Box>
+		</>
```
Comment (fzaninotto) [inline packages/ra-auth-cognito/Readme.md:164]: Same

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: packages/ra-auth-cognito/Readme.md (hunk 1 above)
Comment (adguernier) [reply packages/ra-auth-cognito/Readme.md:127]: I just opened this file on my IDE and the indentation looks correct. I think the weird indentation comes from the Github diff viewer

Comment (fzaninotto) [reply packages/ra-auth-cognito/Readme.md:127]: right, I noticed the same

**Review by fzaninotto — APPROVED** [review]

**Pull request merged**